messaging/src/test/java/org/axonframework/commandhandling/distributed/ConsistentHashTest.java

```diff
@@ -109,4 +110,14 @@ void testConflictingHashesDoNotImpactMembership() {
         assertEquals(member1.name(), consistentHash.getMembers().iterator().next().name());
         assertEquals(consistentHash.getMembers(), consistentHashModified.getMembers());
     }
+
+    @Test
+    void testEquals1() throws Exception {
+        ConsistentHash consistentHash = new ConsistentHash((s) -> "fixed").with(member1, 1, AcceptAll.INSTANCE);
+        ConsistentHash consistentHashModified = consistentHash
+                                                            .with(member2, 1, AcceptAll.INSTANCE)
+                                                            .without(member2);
+        Assertions.assertFalse(consistentHashModified.equals(new ConsistentHash()));
```

Comment:
Nit: This is a bit of a flavor adjustment, but I would expect a static import to be used for `assertFalse`.
Mainly because this is in line with the other test cases in this test class.

Reply:
Lovely adjustment; thank you.